### PR TITLE
feature/export named helper functions

### PR DESCRIPTION
**Pull request by evdtempel**

### CODE REVIEW

File: src/stories/helpers/add-notes.ts [hunk 1]
```diff
@@ -1,5 +1,5 @@
-export default (notes) => {
```
Comment (dan-entous) [inline src/stories/helpers/add-notes.ts]: definitely a good idea. would this, `export const addNotes = (notes) => {`, be better?

File: src/stories/helpers/add-notes.ts [hunk 2]
```diff
@@ -1,5 +1,5 @@
...
+export default function addNotes(notes) {
     return {
         notes,
     };
```
Comment (dan-entous) [inline src/stories/helpers/add-notes.ts]: my tslint config says it’s missing a semicolon.

Comment (evdtempel) [reply src/stories/helpers/add-notes.ts]: my tslint config says it’s an unnecessary semicolon :P

Comment (dan-entous) [reply src/stories/helpers/add-notes.ts]: personally i prefer without semicolons. it seems that the tslint config is conflicting with different settings ... something else to sort out later.

**Pull request closed without merging**